### PR TITLE
adds subsites to cabled array

### PR DESCRIPTION
**Pull request by maka-io**
@DanielJMaher

### CODE REVIEW

File: ooiui/static/js/models/asset_management/PlatformModel.js [hunk 1]
```diff
@@ -106,8 +106,8 @@ var PlatformCollection = Backbone.Collection.extend({
     },
     byArray: function(array) {
         var filtered = this.filter(function (platform) {
-            if (platform.get('reference_designator') !== "" && platform.get('reference_designator').length === 8 ||
-                platform.get('reference_designator').indexOf('GL') > -1) {
+            if (platform.get('reference_designator') !== "" && (platform.get('reference_designator').length === 14 ||
+                platform.get('reference_designator').indexOf('GL') > -1) || platform.get('reference_designator').length === 8) {
```
Comment (DanielJMaher) [inline ooiui/static/js/models/asset_management/PlatformModel.js]: I believe that this line of code is causing the PR to be unmergeable

**Comment by maka-io** [comment]
@DanielJMaher it's a mismatch in the ref, just needed an update. all set!

**Comment by DanielJMaher** [comment]
@maka-io OBE by #752 ?

**Comment by maka-io** [comment]
#hacktoberfest

**Comment by DanielJMaher** [comment]
@oceanzus This does have some elements that we will need, but it was never fully finished (lack of info from MIO).  We will have to revisit at some point, but not yet.

**Comment by oceanzus** [comment]
OBE, closing.

**Pull request closed without merging**